tests/phpunit/tests/rest-api/rest-themes-controller.php

```diff
@@ -388,6 +388,7 @@ public function test_get_item_schema() {
 		$this->assertArrayHasKey( 'align-wide', $theme_supports );
 		$this->assertArrayHasKey( 'automatic-feed-links', $theme_supports );
 		$this->assertArrayHasKey( 'block-templates', $theme_supports );
+		$this->assertArrayHasKey( 'block-template-parts', $theme_supports );
```

Comment:
```suggestion
		$this->assertArrayHasKey( 'block-template-parts', $theme_supports, "Theme supports should have 'block-template-parts' key"  );
```

Add assertions message when a test has multiple assertions, i.e. per Core's testing standards.

Adding messages to other assertions in this test method are out-of-scope for this PR.